dev/extra-ml-requirements.txt

```
@@ -42,3 +42,14 @@ pyspark
 shap
 # Required by mlflow.paddle
 paddlepaddle
+# Required by mlflow.prophet
+Cython
+convertdate
+LunarCalendar
+holidays
+tqdm
+cmdstanpy
+setuptools-git
+python-dateutil
+pystan~=2.19.1.1
+prophet==1.0.1
```

```suggestion
prophet
```

Can we unpin `prophet`?

---

changed